### PR TITLE
Redesign part of account-related layout (sign in, sing up, change password form)

### PR DESCRIPTION
**Pull request by micmarty**
Integrate already written authentication C# code with Material Design front.
TODO Refactor all the pages that are not done yet.
TODO Found bug #19

### CODE REVIEW

**Review by adzka95 — APPROVED** [review]

File: GameApplication/GameApplication/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -1,35 +1,4 @@
-﻿@{
-    ViewData["Title"] = "Home Page";
-}
-
-<script>
-    $(document).ready(function () {
-        startChat();
```
Comment (p-kwiatkowski) [inline GameApplication/GameApplication/Views/Home/Index.cshtml:7]: I think you should call this function in GeneralChat/Index.cshtml

File: GameApplication/GameApplication/Startup.cs [hunk 2]
```diff
@@ -82,11 +87,15 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
...
+                
+                routes.MapRoute(
+                    name: "GeneralChat",
+                    template: "{controller=GeneralChat}/{action=Index}");
             });
 
             app.UseSignalR(routes =>
             {
-                routes.MapHub<ChatRoom>("chat");
+                routes.MapHub<ChatRoom>("general_chat");
```
Comment (p-kwiatkowski) [inline GameApplication/GameApplication/Startup.cs]: You have to update chat.js file. Consider passing chatName to startChat function.

**Review by p-kwiatkowski — CHANGES_REQUESTED** [review]

File: GameApplication/GameApplication/Views/Home/Index.cshtml (hunk 1 above)
Comment (micmarty) [reply GameApplication/GameApplication/Views/Home/Index.cshtml:7]: Requested changes applied

**Review by MateuszSz — APPROVED** [review]

**Review by michalkurzelewski — APPROVED** [review]

**Review by p-kwiatkowski — APPROVED** [review]

**Comment by micmarty** [comment]
GET TO THE CHOPPER!

**Pull request merged**